lib/template.html

```diff
@@ -97,6 +97,8 @@
   {% assign seo_page_image = seo_page_image | escape %}
 {% endif %}
 
+{% assign seo_page_locale = page.lang | default: site.lang | default: "en_US"" %}
```

Comment:
Is there a reason we need to default to `"en_US"` instead of just not setting anything if the user didn't ask for it?